docs/system-settings/dmx-output-mapping.md

```diff
@@ -41,22 +41,32 @@ lines.
 
 ![DMX Settings Window](/docs/images/DMX-Settings-Window.png)
 
-You can individually assign Nodes to Lines using the arrow button, or
-assign all Nodes of the same type using the group assign button. For
-example, to assign each XLR output socket on the back of the console to
-a DMX line, you would click on the Group Assign button of the ExpertDMX
-nodes.
+To assign a node to a line click on the node on the left hand side (e.g.
+on **Expert DMX A**) and then on the line (right hand side) you want this 
+node to assign to. The node will disappear on the left hand side as it is 
+not unassigned any more, and will show up on the right hand side under 
+the line it is assigned to.
 
 ![Node in DMX Settings Window](/docs/images/Node-in-DMX-Settings-Window.png)
 
-To assign a node to a line, click on the **arrow on the node** (the node
-will be highlighted), then click the line you want to assign to it.
-
 To delete a node from the DMX lines, click on the **X** button. To delete
-all nodes from a line, click on the **Group delete button**.
-
-You can show information about the DMX nodes, or the DMX lines, by
-clicking the **i** button. For Ethernet/network type nodes, this allows you
+all nodes from a line, click on the **Group delete button** (the **X** button 
+in the very Dmx Line). To delete all nodes from all lines, click on the **X** 
+button close to the header Dmx Lines.
```

Comment:
```suggestion
in the very DMX Line). To delete all nodes from all lines, click on the **X** 
button close to the header DMX Lines.
```

This is what is causing the test to fail

Reply:
Thanks

Reply:
It need to be **DMX** not **Dmx**, so unfortunately de88164 won't pass either

Reply:
"very DMX line" isn't the right way to say it, I think it would be best to say "the X button next to the DMX line". 

A picture might be good here as there are a lot of different X buttons...

Reply:
Thanks again. I hope I don't keep you too busy :)

Reply:
My dear british comrades, you are always welcome to correct my wording - I know that my writing probably isn't perfect. Actually I do all this while checking the **german** translation...

Reply:
> Thanks again. I hope I don't keep you too busy :)

Thankfully there are long VTs on this show